### PR TITLE
./scripts/fix.sh: Takes care of goimports across the whole project.

### PR DESCRIPTION
**Pull request by ptabor**

### CODE REVIEW

File: server/config/config.go [hunk 1]
```diff
@@ -22,15 +22,17 @@ import (
...
+
 	"go.etcd.io/etcd/client/pkg/v3/transport"
 	"go.etcd.io/etcd/client/pkg/v3/types"
 	"go.etcd.io/etcd/pkg/v3/netutil"
 	"go.etcd.io/etcd/server/v3/etcdserver/api/v3discovery"
 	"go.etcd.io/etcd/server/v3/storage/datadir"
-	"go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"
 
-	bolt "go.etcd.io/bbolt"
 	"go.uber.org/zap"
```
Comment (ahrtr) [inline server/config/config.go:33]: same as above

File: server/embed/config.go [hunk 2]
```diff
@@ -41,12 +41,13 @@ import (
 	"go.etcd.io/etcd/server/v3/etcdserver/api/v3compactor"
 	"go.etcd.io/etcd/server/v3/etcdserver/api/v3discovery"
 
-	bolt "go.etcd.io/bbolt"
 	"go.uber.org/multierr"
 	"go.uber.org/zap"
 	"golang.org/x/crypto/bcrypt"
 	"google.golang.org/grpc"
 	"sigs.k8s.io/yaml"
```
Comment (ahrtr) [inline server/embed/config.go:48]: ditto

File: server/etcdserver/api/etcdhttp/health.go [hunk 3]
```diff
@@ -21,13 +21,14 @@ import (
...
 	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+
 	"go.etcd.io/etcd/api/v3/etcdserverpb"
 	pb "go.etcd.io/etcd/api/v3/etcdserverpb"
 	"go.etcd.io/etcd/client/pkg/v3/types"
 	"go.etcd.io/etcd/server/v3/auth"
 	"go.etcd.io/etcd/server/v3/config"
 	"go.etcd.io/raft/v3"
-	"go.uber.org/zap"
```
Comment (ahrtr) [inline server/etcdserver/api/etcdhttp/health.go:30]: I am curious why such kind of import disorder isn't discovered by https://github.com/etcd-io/etcd/blob/a4c1b3a9e2b32d64a3685ab35a39cbb62b3e08c7/.golangci.yaml#L33 ?

@demoManito

Comment (demoManito) [reply server/etcdserver/api/etcdhttp/health.go:30]: I need to study it.

File: etcdutl/etcdutl/backup_command.go [hunk 4]
```diff
@@ -37,8 +38,9 @@ import (
 	"go.etcd.io/etcd/server/v3/verify"
 	"go.etcd.io/raft/v3/raftpb"
 
-	bolt "go.etcd.io/bbolt"
 	"go.uber.org/zap"
```
[the remark this answers is not in the document]
Comment (ptabor) [reply etcdutl/etcdutl/backup_command.go:41]: done

File: server/embed/config.go (hunk 2 above)
Comment (ptabor) [reply server/embed/config.go:48]: done

File: server/etcdserver/api/etcdhttp/health.go (hunk 3 above)
Comment (ahrtr) [reply server/etcdserver/api/etcdhttp/health.go:30]: Thank you @demoManito . Please feel free to deliver a PR to fix it if you have any findings.

**Review by ahrtr — APPROVED** [review]
LGTM

File: server/etcdserver/api/etcdhttp/health.go (hunk 3 above)
Comment (demoManito) [reply server/etcdserver/api/etcdhttp/health.go:30]: OK

**Pull request merged**